community/community-it/it-test-support/src/main/java/org/neo4j/bolt/transport/Neo4jWithSocket.java

```diff
@@ -53,28 +49,18 @@
 import static org.neo4j.configuration.connectors.BoltConnector.EncryptionLevel.OPTIONAL;
 import static org.neo4j.test.rule.TestDirectory.testDirectory;
 
-public class Neo4jWithSocket extends ExternalResource
+public class Neo4jWithSocket
 {
-    private final Supplier<FileSystemAbstraction> fileSystemProvider;
```

Comment:
I reverted this, our convention is that everything that can be `final` should be declared so.